dahdi-tools.spec

```diff
@@ -51,6 +51,7 @@ Requires(pre):   %{_sbindir}/groupadd
 Conflicts:       zaptel-utils
 Obsoletes:       libtonezone
 Provides:       libtonezone
+Provides:       libtonezone.so.2()(64bit)
```

Comment:
Which package is requesting it?

Reply:
I was about saying asterisk18-dahdi, but I tried it again to be sure and I'm no more able to reproduce it...